tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "moduleResolution": "node",
     "noEmit": true,
     "noErrorTruncation": true,
+    "skipLibCheck": true,
```

Comment:
Hmm. It's a shame to disable type checking for libraries. I see that you found [an error that there is no apparently solution for](https://github.com/isaacs/node-lru-cache/issues/292#issuecomment-1524256758) though. I guess we can use this as a workaround in the meantime.